R/return_linter.R

```diff
@@ -0,0 +1,206 @@
+#' Return linter
+#'
+#' This linter checks for explicit [return()] at the end of a function
+#'
+#' @param use_implicit_returns Whether to use implicit or explicit returns
+#'
+#' @param additional_allowed_func Names of additional functions that are
+#'  accepted as return if `!use_implicit_returns`
+#'
+#' @param additional_side_effect_func Names of additional functions that are
```

Comment:
Maybe call this `except`?

Doc could be:
Names of functions that do not need an explicit return.

Reply:
I would add an “additional” to the doc, but otherwise I think it’s a good idea.

Reply:
Okay. Additional to what should then also be documented, i.e. the built-in list of exceptions.

Reply:
Handled in latest batch of commits, PTAL.